jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
I wish we didn't have to maintain parent_id on the clients in the first place. It's only needed for compatibility with Zipkin's single span per RPC model that we still optionally support.

I think I'm ok with keeping both parent_id & child_of reference and just adding a sanitizer to the backend to normalize the representation.